tests/property/ic_api/candid/test/tests.ts

```diff
@@ -1,110 +1,78 @@
-import { ActorSubclass } from '@dfinity/agent';
-import { defaultPropTestParams, expect, it, Test } from 'azle/test';
+globalThis._azleExperimental = true;
+import { execSyncPretty } from 'azle/src/build/stable/utils/exec_sync_pretty';
```

Comment:
execSyncPretty should be used everywhere in this file right?

Reply:
Let's not use execSyncPretty in our tests moving forward

Reply:
done